docs/usage/PRODUCT_SYNC.md

```diff
@@ -47,15 +47,15 @@ order for the sync to resolve the actual ids of those references, those `key`s h
          ````
      
 3. It is an important responsibility of the user of the library to instantiate a `sphereClient` that has the following properties:
-    - Limits the amount of concurrent requests done to CTP. This can be done by decorating the `sphereClient` with 
+    - Limits the number of concurrent requests done to CTP. This can be done by decorating the `sphereClient` with 
    [QueueSphereClientDecorator](http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/client/QueueSphereClientDecorator.html) 
     - Retries on 5xx errors with a retry strategy. This can be achieved by decorating the `sphereClient` with the 
    [RetrySphereClientDecorator](http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/client/RetrySphereClientDecorator.html)
    
-   You can use the same client instantiating used in the integration tests for this library found 
+   You can instantiate the client the same way it is instantiated in the integration tests for this library found
```

Comment:
You probably missed to do this same change in the CATEGORY_SYNC.md doc.

Reply:
nice catch!  https://github.com/commercetools/commercetools-sync-java/pull/319/commits/2c2b61de5ac9d3d91b5f139a71ae8c13775c2e9a